stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -89,6 +108,11 @@ public type Client client object {
     #            in the batch have executed successfully, the error will be `nil`. If one or more commands have failed,
     #            the `returnedError` field will give the corresponding `Error` along with the int[] which
     #            contains updated row count or the status returned from each command in the batch.
+    #
+    # # Deprecated
+    # New experimental jdbc2 module is introduced in 1.2 release and it will be fully supported with in 1.3 release
```

Comment:
```suggestion
    # A new experimental JDBC2 module is introduced in the Ballerina 1.2 release and it will be fully supported by the 1.3 release.
```